### DISCUSSION TITLE
having issues with bed leveling using the 1.3.3 10 x 10 ubl bl for 4.2.2 on a 400 x 400 x 450 bed for 3v2

### DISCUSSION THREAD

**Discussion by blackoutlaw** [Jyers/Marlin#1063] [Q&A] [answered]
ok i finally got the upgrade to bed size done and flashed the above 10 x 10 ubl firmware all seams to be doing good except after i do the generate a mesh it only does about 60 points and then says that it got all accessible points and it does a random pattern when doing the mesh and when viewing the mesh it is not consistent then i do the auto tilt and it gets worse as in it causes some points not to be within the .1 of the bed and others to drive the nozzle into the bed. is there an issue with the firmware or is it a physical thing i have done all the leveling via the lcd screen menues . could someone let me know if i am doing anything wrong. at this point i have not changed my start gcode in cura so if there is something i need to do that would be a great help. as my bed is now a 400 x 400x 450 from the original 3v2 bed size. thanks for any help

**Comment by smooreace** [comment]
Is there a reason to post multiple times?

**Comment by blackoutlaw** [comment]
thought were i posted before was a closed thread sorry

**Reply by smooreace** [reply]
Im not sure, not going to invest the time to go find it again. But posting to a closed thread re-opens it. So either way.

**Comment by Jyers** [comment, marked as answer]
See #474 for why not all points are probed.

As far as your real issues, sounds like a hardware issue of some kind. Has it improved at all or are you still facing issues?
